src/generic_rosenbrock.jl

```diff
@@ -0,0 +1,59 @@
+struct RosenbrockTableau
+    a::Array{Real,2}
```

Comment:
Parameterize the eltype instead of using Real, so that the compiler can specialize in it.